.github/workflows/build.yml

```diff
@@ -77,8 +77,7 @@ jobs:
         if: matrix.target == 'darwin'
         shell: bash
         run: |
-          # install wails with amd64 arch, even for arm64, to be able to use it for the build
-          GOARCH=amd64 go install github.com/wailsapp/wails/v2/cmd/wails@v2.6.0
+          export PATH=$PATH:~/go/bin/darwin_amd64
```

Comment:
We remove the installation of wails ? 🤔

Reply:
it's already installed in task install, in action install.yaml